box-browse-sdk/src/main/res/layout/box_browsesdk_fragment_browse.xml

```diff
@@ -30,4 +30,11 @@
         android:layout_gravity="center"
         android:indeterminate="true"
         android:visibility="gone"/>
+
+    <ListView
```

Comment:
Can you create a separate layout rather than combine it into the browse fragment. I don't mind duplicating the layout, but would rather avoid having the browse fragment deal with this.